Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.6
+
+WORKDIR /app
+
+RUN wget https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh -P /usr/local/bin \
+    && chmod +x /usr/local/bin/wait-for-it.sh
+
+# Keeps Python from generating .pyc files in the container
+ENV PYTHONDONTWRITEBYTECODE 1
+
+# Turns off buffering for easier container logging
+ENV PYTHONUNBUFFERED 1
+
+# Install pip requirements
+ARG REQUIREMENTS=requirements.txt
+COPY requirements.txt dev_requirements.txt /app/
+
+RUN pip install -r $REQUIREMENTS --disable-pip-version-check
+
+ADD . /app
+
+EXPOSE 8000
```

Comment:
Move the `EXPOSE` line up to slightly speed up the build process, as it won't be invalidated by code changes. As this won't change at all, I'd put it even before the `ARG REQUIREMENTS` line

Reply:
done